website/docs/reference/resource-configs/snowflake-configs.md

```diff
@@ -346,94 +346,135 @@ In the configuration format for the model SQL file:
+    [+](/reference/resource-configs/plus-prefix)on_configuration_change: apply | continue | fail
+    [+](/reference/resource-configs/plus-prefix)[target_lag](#target-lag): downstream | <time-delta>
+    [+](/reference/resource-configs/plus-prefix)[snowflake_warehouse](#configuring-virtual-warehouses): <warehouse-name>
+```
 
-The following examples create a dynamic table:
+</File>
 
-<File name='models/YOUR_MODEL_NAME.sql'>
+</TabItem>
 
-```sql
-{{ config(
-    materialized = 'dynamic_table',
-    snowflake_warehouse = 'snowflake_warehouse',
-    target_lag = '10 minutes',
-) }}
-```
 
-</File>
+<TabItem value="property-yaml">
 
-<File name='dbt_project.yml'>
+<File name='models/properties.yml'>
 
 ```yaml
+version: 2
+
 models:
-  path:
-    materialized: dynamic_table
-    snowflake_warehouse: snowflake_warehouse
-    target_lag: '10 minutes'
+  - name: [<model-name>]
+    config:
+      [materialized](/reference/resource-configs/materialized): dynamic_table
+      on_configuration_change: apply | continue | fail
+      [target_lag](#target-lag): downstream | <time-delta>
+      [snowflake_warehouse](#configuring-virtual-warehouses): <warehouse-name>
 ```
 
 </File>
 
-### Monitored configuration changes
+</TabItem>
+
 
-The settings below are monitored for changes applicable to `on_configuration_change`.
+<TabItem value="config">
 
-#### Target lag
+<File name='models/<model_name>.sql'>
 
-Changes to `target_lag` can be applied by running an `ALTER` statement. Refreshing is essentially
-always on for dynamic tables; this setting changes how frequently the dynamic table is updated.
+```jinja
+{{ config(
+    [materialized](/reference/resource-configs/materialized)="dynamic_table",
+    on_configuration_change="apply" | "continue" | "fail",
+    [target_lag](#target-lag)="downstream" | "<integer> seconds | minutes | hours | days",
+    [snowflake_warehouse](#configuring-virtual-warehouses)="<warehouse-name>",
+) }}
+```
 
-#### Warehouse
+</File>
+
+</TabItem>
+
+</Tabs>
 
-Changes to `snowflake_warehouse` can be applied via an `ALTER` statement.
+Find more information about these parameters in the Snowflake docs:
+- [CREATE DYNAMIC TABLE](https://docs.snowflake.com/en/sql-reference/sql/create-dynamic-table)
+
+### Target lag
+
+Snowflake allows two configuration scenarios for scheduling automatic refreshes: 
+- **Time-based** &mdash; Provide a value of the form `<int> { seconds | minutes | hours | days }`. For example, if the dynamic table needs to be updated every 30 minutes, use `target_lag='30 minutes'`.
+- **Downstream** &mdash; For scenarios where the referenced objects are themselves dynamic tables, it might be desirable to refresh the dynamic table whenever the underlying dynamic table is refreshed. In this scenario, use `target_lag='downstream'`. This allows for refresh schedules to be controlled once, at the source, instead of at each layer.
```

Comment:
@mikealfare Should we adopt the Snowflake docs language - downstream rather than dependency-based?

Reply:
Done